src/main/java/appeng/mixins/AbstractContainerScreenAccessor.java

```diff
@@ -0,0 +1,15 @@
+package appeng.mixins;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+import net.minecraft.world.inventory.Slot;
+
+@Mixin(AbstractContainerScreen.class)
+public interface AbstractContainerScreenAccessor {
```

Comment:
We tend to prefer access wideners (the equivalent of access transformers on fabric).

Reply:
done